doc/source/contribution/documentation/introduction.rst

```diff
@@ -13,10 +13,11 @@ Groonga uses Sphinx_ as documentation tool.
 
 .. _Sphinx: https://www.sphinx-doc.org/
 
-Here are command lines to install Sphinx.
+Here are command lines to install Sphinx and the other dependencies.
 
 Debian GNU/Linux, Ubuntu::
 
+  % ./setup.sh
```

Comment:
I will add it  because I'm not sure which is merged earlier although I've already added it at the following PR.
- https://github.com/groonga/groonga/pull/1712